### PR TITLE
Prevent mass assignment

### PR DESCRIPTION
**Pull request by penestia**

### CODE REVIEW

File: models/group.rb [hunk 1]
```diff
@@ -3,13 +3,17 @@
 require 'json'
 require 'sequel'
 require_relative 'init'
+
+
```
Comment (atellezsazo) [inline models/group.rb]: Did you check rubocop?

File: models/problem.rb [hunk 2]
```diff
@@ -10,6 +10,11 @@ class Problem < Sequel::Model
     many_to_one :group
 
     plugin :timestamps
+    plugin :whitelist_security
+    set_allowed_columns :description 
+    plugin :uuid, field :id
+
+
```
Comment (atellezsazo) [inline models/problem.rb]: blank lines

File: specs/problems_spec.rb [hunk 3]
```diff
@@ -46,6 +46,9 @@
     _(last_response.status).must_equal 404
   end
 
+
+
```
Comment (atellezsazo) [inline specs/problems_spec.rb]: =

File: models/group.rb (hunk 1 above)
Comment (penestia) [reply models/group.rb]: No haha

**Pull request merged**